static-analysis/StaticAnalysisChapter.md

```diff
@@ -404,16 +403,15 @@ function loopcontents(e::Expr)
   end
   lines
 end
-~~~
+~~~~
 
 And now to explain in pieces:
 
-1. 
 ~~~.jl
 b = body(e)
 ~~~
 
-    We start by getting all the expressions in the body of method, as an `Array`. `body` is a function that I've already implemented:
+We start by getting all the expressions in the body of method, as an `Array`. `body` is a function that I've already implemented:
 
     ~~~.jl
```

Comment:
Is the indentation important here? It's getting lost in the layout, but if it's not relevant that's okay.